src/Faker/Container/ContainerInterface.php

```diff
@@ -6,8 +6,4 @@
 
 interface ContainerInterface extends BaseContainerInterface
 {
-    /**
-     * Get the bindings between Extension interfaces and implementations.
-     */
-    public function getDefinitions(): array;
```

Comment:
Given that the `ContainerInterface` interface would now be empty, perhaps we should remove it?

Reply:
No harm is maintaining our own extension of it, in case it's needed.